migration/gauss_hit_finder/register_find_hits_with_gaussians_design1.cpp

```
@@ -0,0 +1,101 @@
+  auto const layer_unfold_input = config.get<std::string>("layer_unfold_input");
+  auto const layer_unfold_output = config.get<std::string>("layer_unfold_output");
+
+  m.unfold<examples::unfold_wire_vector_design1>("unfold_wire_vector_design1",
+                                                 &examples::unfold_wire_vector_design1::predicate,
+                                                 &examples::unfold_wire_vector_design1::unfold,
+                                                 layer_unfold_output,
+                                                 concurrency::unlimited)
+    .input_family(product_selector{.creator = "wires", .layer = layer_unfold_input, .suffix = ""});
+
+  examples::find_hits_with_gaussians_design1_cfg cfg = {
+    .filter_hits = config.get<bool>("filter_hits"),
```

Cppcheck (2.21.0)

[style] The function 'isValidChannelID' is never used.

(unusedFunction)